src/etc/inc/certs.inc

```diff
@@ -408,11 +415,15 @@ function csr_generate(& $cert, $keylen, $dn, $type = "user", $digest_alg = "sha2
 
 	$args = array(
 		"x509_extensions" => $cert_type,
-		"req_extensions" => "req_{$cert_type}",
```

Comment:
Why is this line removed? It doesn't appear to be related to the other changes.

Reply:
fixed
typo